data/mods/DinoMod/monsters/juvenile.json

```diff
@@ -467,6 +506,15 @@
     "color": "light_green_magenta",
     "upgrades": { "age_grow": 365, "into": "mon_pentaceratops" }
   },
+  {
+    "id": "mon_kosmoceratops_juvenile",
+    "type": "MONSTER",
+    "name": "green and magenta four-legged juvenile",
+    "copy-from": "mon_gallimimus_juvenile",
+    "description": "A small four-legged plant-eating dinosaur juvenile with huge shiny eyes, it could be from a number of different species.",
```

Comment:
```suggestion
    "description": "A small, four-legged, plant-eating dinosaur juvenile with huge shiny eyes, it could be from a number of different species.",
```